Content.Server/Storage/EntitySystems/StorageSystem.Fill.cs

```diff
@@ -39,8 +50,42 @@ private void OnStorageFillMapInit(EntityUid uid, StorageFillComponent component,
                 Logger.ErrorS("storage", $"Tried to StorageFill {entry.PrototypeId} inside {uid} but can't.");
                 EntityManager.DeleteEntity(ent);
             }
+        }
+
+        // handle orgroup spawns
+        foreach (var group in orGroupedSpawns)
+        {
+            double diceRoll = _random.NextDouble() * group.Value.CumulativeProbability;
+            _random.Shuffle(group.Value.Entries);
+
+            double cumulative = 0.0;
+            for (int i = 0; i < group.Value.Entries.Count; i++)
```

Comment:
```suggestion
            foreach (var entry in group.Value.Entries)
```

Reply:
yeah, left that one in by mistake, I think I was trying to see if I can select randomly by index.